Modules/_sqlite/connection.c

```diff
@@ -2561,6 +2561,12 @@ set_autocommit(pysqlite_Connection *self, PyObject *val, void *Py_UNUSED(ctx))
     return 0;
 }
 
+static PyObject *
+get_sig(pysqlite_Connection *self, void *Py_UNUSED(ctx))
```

Comment:
```suggestion
get_sig(PyObject *self, void *Py_UNUSED(ctx))
```
And you will not need the cast.